internal/pkg/config/model.go

```diff
@@ -87,3 +95,18 @@ type RedisConfig struct {
 type BlobConfig struct {
 	MaxInlineSize int
 }
+
+// GRPCConfig Holds general configuration for grpc server
+type GRPCConfig struct {
```

Comment:
Please rename to GrpcServerConfig and put the fields directly in the struct. Also, please define the values as time.Duration to align with the other config properties. I.e.

type GrpcServerConfig struct {
	MaxConnectionIdle            time.Duration
	MaxConnectionAge            time.Duration
	MaxConnectionAgeGrace time.Duration
	Time                                    time.Duration
	Timeout                              time.Duration
}